tests/subsys/fs/nvs/boards/native_posix.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&flash0 {
+	erase-block-size = <0x400>;
+};
```

Comment:
Is this needed for the PR ?

Reply:
The test assumes such a sector size, and otherwise works fine for native_posix platform. I used the native_posix platform while developing the cache and thought it would be nice to make the test work on that platform. I can extract it to a separate PR if you would prefer so.